public/pages/Findings/components/FindingsTable/FindingsTable.tsx

```diff
@@ -205,7 +206,19 @@ export default class FindingsTable extends Component<FindingsTableProps, Finding
         name: 'Rule severity',
         sortable: true,
         dataType: 'string',
-        render: (ruleSeverity: string) => capitalizeFirstLetter(ruleSeverity) || DEFAULT_EMPTY_DATA,
+        align: 'left',
+        render: (ruleSeverity: string) => {
+          const severity = capitalizeFirstLetter(ruleSeverity) || DEFAULT_EMPTY_DATA;
+
+          return (
+            <EuiBadge
+              color={getSeverityColor(severity).background}
+              style={{ color: getSeverityColor(severity).text }}
```

Comment:
Nitpick: Similar to the comment above, maybe considering slightly refactoring this to store the `getSeverityColor` output in a variable, and call the variable instead of the function multiple times.